airflow/www/utils.py

```diff
@@ -127,13 +128,22 @@ def get_mapped_summary(parent_instance, task_instances):
     }
 
 
-def encode_ti(
-    task_instance: Optional[TaskInstance], is_mapped: Optional[bool], session: Optional[Session]
-) -> Optional[Dict[str, Any]]:
+def encode_ti(dag_run: DagRun, task, session: Session) -> Optional[Dict[str, Any]]:
```

Comment:
I feel it may be better to either rename this function, or move logic obtaining the ti out of this function.

Reply:
Maybe `get_task_summary`?

Reply:
Works for me!